README.md

```diff
@@ -500,6 +500,32 @@ type Account struct {
     Name string `json:"name" example:"account name"`
 }
 ```
+
+### Model composition in response
```

Comment:
Also need to add a hyperlink in Content.Examples at the head of README.md

Reply:
sure, linked added.